packages/workers/src/scripts/il-alerts-discord.ts

```diff
@@ -0,0 +1,139 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+import BigNumber from 'bignumber.js';
+import Discord from "discord.js";
```

Comment:
You should add this to the worker's package.json - `cd packages/worker && yarn add discord.js`

Reply:
will do.

Reply:
@kkennis pushed discord.js into the package.json.